### PR TITLE
First time commiter improvements

### PR DESCRIPTION
**Pull request by bltavares**
This PR adds a few minor improvements for first time contributors on the project, so there is a nice experience on a fresh checked out repository.

Each commit has detailed messages on the pitfalls I faced when I first tried to build and test locally.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,8 +219,27 @@ add your name and email address to the
 under the "Contributors" section using the format:
 `First Last <email@example.com>`.
 
+For first timer contributors to a Go project, there is how to get the project setup.
```
Comment (cv) [inline README.md]: `To set up the project for local development, ensure that the $GOPATH variable…`

I'm not a first-time contributor, and I had issues with this too :)

File: README.md [hunk 2]
```diff
@@ -219,8 +219,27 @@ add your name and email address to the
...
+First, ensure that `$GOPATH` variable is pointing to an existing directory,
+where all your Go code will be available.
+
+After forking your repo on Github, execute the following to have the project
+setup.
```
Comment (cv) [inline README.md]: `s/execute the following to have the project setup./run:/`

File: README.md [hunk 3]
```diff
@@ -219,8 +219,27 @@ add your name and email address to the
...
+go get github.com/digitalocean/do-agent
+cd $GOPATH/src/github.com/digitalocean/do-agent
+
+git remote add my-fork <the url for your fork>
+make test # ensure tests are passing
+```
+
 Finally, submit a pull request for review!
 
+```sh
```
Comment (cv) [inline README.md]: Should mention that you need to push before submitting the PR?

Comment (bltavares) [inline README.md]: 👍

**Comment by bltavares** [comment]
@cv the requested changes have been applied. Let me know if there is any thing else left to adjust.

**Comment by utlemming** [comment]
Callling this LGTM and merging.

**Pull request merged**